lib/checkleakautovar.cpp

```diff
@@ -206,6 +206,30 @@ static bool isPointerReleased(const Token *startToken, const Token *endToken, un
     return false;
 }
 
+/** checks if token is a name of a function in a function call:
+    func(arg) 
+ or func<templ_arg>(arg)
+
+    @return opening parenthesis token or NULL if not a function call
+*/
```

Comment:
This doxygen comment looks weird to me. we typically start each line with a `* ` like:

    /** checks if nameToken is a name of a function in a function call:
     *     func(arg)
     * or
     *     func<temp1_arg>(arg)
     * @param nameToken Function name token
     * @return opening parenthesis token or NULL if not a function call
     */